ansible_base/jwt_consumer/common/auth.py

```diff
@@ -13,6 +13,7 @@
 from ansible_base.jwt_consumer.common.cert import JWTCert, JWTCertException
 from ansible_base.lib.utils.auth import get_user_by_ansible_id
 from ansible_base.lib.utils.translations import translatableConditionally as _
+from ansible_base.rbac.models import RoleUserAssignment
```

Comment:
I think this is unsafe to import at the top of the file. Elsewhere in this file `RoleDefinition` is imported in-line.

https://github.com/ansible/django-ansible-base/blob/2f8fcf18b95f975963fec893592ec33506421f78/ansible_base/jwt_consumer/common/auth.py#L195

The concern is that ansible/galaxy_ng is using the jwt_consumer app but is not using the RBAC app. Importing a model from an app not in INSTALLED_APP will error.